_overview/fiat-permissions-overview.md

```diff
@@ -0,0 +1,240 @@
+
+
+## Mapping exercise
+
+Answer the following questions to figure out how to map roles and permissions in your deployment:
+
+* Which roles/groups have READ and/or WRITE access to which Clouddriver accounts
+* Which roles/groups have READ, WRITE, EXECUTE access to each Spinnaker Application
+* Which roles/groups have READ and/or WRITE/EXECUTE access to which CI/Build accounts
+
+The following image shows an example result of this exercise based on the user roles described in [Example Fiat](#example-fiat):
+
+![Mapping Exercise Role Matrix](/images/fiat_overview_role_matrix.png)
+
+## Example Configurations
+
+The following sections describe some of the roles from the role matrix example in the [Mapping exercise](#mapping-exercise).
+
+## Superuser
+
+`fiat-admin` is the superuser and has permissions across your whole Spinnaker deployment.
+
+The configuration for `fiat-admin` in the `fiat-local.yml` file looks like the following snippet:
+
+```
+admin:
+  roles:
+    - fiat-admin
+```
+## Infrastructure 
+
+`dev-infra` is one of the potential deployment targets (pictured in ([Mapping exercise](#mapping-exercise)). 
+
+The Halconfig snippet for configuring access to `dev-infra` based on our mapping exercise looks similar to the following:
+
+```
+accounts:
+- name: dev-infra
+  permissions:
+    READ:
+      - admin
+      - dev
+      - qa
+      - ops
+    WRITE
+      - admin
+      - dev
+      - ops
+```
+
+Based on which roles have what access to other infrastructure accounts, the configuration looks different.
+
+Note that `fiat-admin` does not need to be explicitly granted permissions. Every other user role must be granted permissions explicitly. 
+
+For information about how to configure permissions for Clouddriver accounts, see [Accounts](https://www.spinnaker.io/setup/security/authorization/#accounts).
+
+## Continuous integration system
+
+`build1` is a Jenkins deployment used for CI in this example. The Halconfig for controlling access to `build1` looks similar to the following snippet:
+
+```
+ci:
+  jenkins:
+    enabled: true
+    masters:
+    - name: build1
+      permissions:
+        READ:
+        - admin
+        - dev
+        - qa
+        - ops
+        WRITE:
+        - admin
+        - ops
+```
+
+## Applications
+
+`app1` is one of the applications that needs to be deployed. Configuring permissions for an application is done in Deck, Spinnaker's UI, when you create or edit an application:
+
+![app1 permissions](/images/fiat_overview_app1_perms.png)
+
+`app2`, `app3`, and `app4` will look slightly different since they have different permissions based on the mapping exercise. 
+
+For information about how to configure permissions for applications, see [Applications](https://www.spinnaker.io/setup/security/authorization/#applications).
+
+## Applying changes
+
+Whenever you make a change to permissions, run `hal deploy apply` to apply your changes to the Spinnaker deployment.
```

Comment:
This only needs to be done if making a change in the Hal config.  For instance, adding a service account will not require the user to do this.